### DISCUSSION TITLE
Cant download AttributeError: 'int' object has no attribute 'strip'

### DISCUSSION THREAD

**Discussion by emilic14** [pyload/pyload#4003] [Help] [answered]
Hello,

trying to download and systematically get this error : 
this is with a direct download.

`AttributeError: 'int' object has no attribute 'strip'`

`
08.08.2021 11:28:24 | WARNING | Download failed: xxxxxxx\| 'int' object has no attribute 'strip'
-- | -- | -- | --
705 | ? | ? | Traceback (most recent call last):
706 | ? | ? | File "/overlay/pyload-0.4.20/module/PluginThread.py", line 189, in run
707 | ? | ? | pyfile.plugin.preprocessing(self)
708 | ? | ? | File "/overlay/pyload-0.4.20/module/plugins/internal/Base.py", line 291, in preprocessing
709 | ? | ? | return self._process(*args, **kwargs)
710 | ? | ? | File "/overlay/pyload-0.4.20/module/plugins/internal/Hoster.py", line 118, in _process
711 | ? | ? | self.process(self.pyfile)
712 | ? | ? | File "/overlay/pyload-0.4.20/module/plugins/hoster/Http.py", line 38, in process
713 | ? | ? | self.download(url, ref=False, disposition=True)
714 | ? | ? | File "/overlay/pyload-0.4.20/module/plugins/internal/Hoster.py", line 300, in download
715 | ? | ? | self.fail(e.args[0])
716 | ? | ? | File "/overlay/pyload-0.4.20/module/plugins/internal/Base.py", line 391, in fail
717 | ? | ? | msg = msg.strip()
718 | ? | ? | AttributeError: 'int' object has no attribute 'strip'
719 | 08.08.2021 11:28:24 | INFO | Debug Report written to debug_BasePlugin_08-08-2021_11-28-24.zip

`

Running pyload v 0.4.20
with python 2.7.18

any ideas of what's going on ?

**Comment by OzzieIsaacs** [comment]
Which hoster you are trying to download from?
The log content isn't displayed properly, please fix this, otherwise it will be very hard to figure out the problem (we only see "?")

**Reply by emilic14** [reply]
Thanks! updated. Its a direct download, but also saw it when trying to use uptobox...
I honestly don't understand the issue...

**Reply by OzzieIsaacs** [reply]
pyload is expecting a string, but getting an integer value. 

The errormessage is to vague. 
There is no context what happend before. I think gammacode will at least need the "debug_BasePlugin_08-08-2021_11-28-24.zip" file and also the link which is causing problems. Otherwise he can't help you

**Comment by emilic14** [comment, marked as answer]
Solved!

So I was actually having a hard drive error... no idea what happen... but changed download folder and its now working without an issue...
go figure...

Thanks!
